notebooks/LST1_observation_simulator.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>I changed the way the background and gamma rate files are read. The initial implementation is not working unless you define in advance LSTCHAIN variable. In this way, that is unnecessary, simply using the functionality lstchain already has.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>